### PR TITLE
CTCP-1297: Endorsement date page

### PR DESCRIPTION
**Pull request by MatthewAlexanderBeck**

### CODE REVIEW

File: app/controllers/incident/EndorsementDateController.scala [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  navigatorProvider: IncidentNavigatorProvider,
+  formProvider: DateFormProvider,
+  actions: Actions,
+  val controllerComponents: MessagesControllerComponents,
+  view: EndorsementDateView
+)(implicit ec: ExecutionContext)
+    extends FrontendBaseController
+    with I18nSupport {
+
+  private val minDate = LocalDate.of(2020: Int, 12: Int, 31: Int) //"31 December 2020"
```
Comment (ajchilts) [inline app/controllers/incident/EndorsementDateController.scala:47]: Should we hold this is config?

File: test/controllers/incident/EndorsementDateControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import views.html.incident.EndorsementDateView
+
+import java.time.{Clock, LocalDate, ZoneOffset}
+import scala.concurrent.Future
+
+class EndorsementDateControllerSpec extends SpecBase with AppWithDefaultMockFixtures {
+
+  private val minDate = LocalDate.of(2020: Int, 12: Int, 31: Int) //"31 December 2020"
```
Comment (ajchilts) [inline test/controllers/incident/EndorsementDateControllerSpec.scala:34]: Less relevant in a test but perhaps still pull from config?

File: test/views/incident/EndorsementDateViewSpec.scala [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import play.api.data.Form
+import play.twirl.api.HtmlFormat
+import views.behaviours.DateInputViewBehaviours
+import views.html.incident.EndorsementDateView
+
+import java.time.{Clock, LocalDate, ZoneOffset}
+
+class EndorsementDateViewSpec extends DateInputViewBehaviours {
+
+  private val minDate = LocalDate.of(2020: Int, 12: Int, 31: Int) //"31 December 2020"
```
Comment (ajchilts) [inline test/views/incident/EndorsementDateViewSpec.scala:30]: As above.

**Review by ajchilts — CHANGES_REQUESTED** [review]

File: app/controllers/incident/EndorsementDateController.scala (hunk 1 above)
Comment (MatthewAlexanderBeck) [reply app/controllers/incident/EndorsementDateController.scala:47]: 100% will add that in now :+1:

**Review by ajchilts — APPROVED** [review]
LGTM 👍

**Pull request merged**